### ISSUE TITLE
binaries not compatible with windows xp

### ISSUE THREAD

**Issue by blueclouds8666** [vxiiduu/NotepadEx#1]
the following error appears while trying to run the program through XP SP3:

![xperror](https://user-images.githubusercontent.com/52018385/148313377-facda9c7-cbf2-477b-a558-20a63375590e.PNG)

**Comment by vxiiduu** [comment]
Yes, I am aware of that. The code uses the TaskDialogIndirect function which is most likely the culprit. Windows XP compatibility will be restored in the next release.

**Comment by vxiiduu** [comment]
Version 1.1.1.0 solves this issue
https://github.com/vxiiduu/NotepadEx/releases/tag/Version1.1.1
![image](https://user-images.githubusercontent.com/73044267/148317547-1e73e313-da6c-49bc-8cca-7670cc142720.png)

**Comment by blueclouds8666** [comment]
good, thanks for supporting xp
